report_xlsx/report/report_xlsx.py

```diff
@@ -17,8 +17,30 @@
 class ReportXlsxAbstract(models.AbstractModel):
     _name = 'report.report_xlsx.abstract'
 
+    def _get_objs_for_report(self, docids, data):
+        """
+        Returns objects for xlx report.  From WebUI these
+        are either as docids taken from context.active_ids or
+        in the case of wizard are in data.  Manual calls may rely
+        on regular context, setting docids, or setting data.
+
+        :param docids: list of integers, typically provided by
+            qwebactionmanager for regular Models.
+        :param data: dictionary of data, if present typically provided
+            by qwebactionmanager for TransientModels.
+        :param ids: list of integers, provided by overrides.
+        :return: recordset of active model for ids.
+        """
+        if docids:
+            ids = docids
+        elif data and 'context' in data:
+            ids = data["context"].get('active_ids', [])
```

Comment:
would it make sense to get `active_model` from `data['context']` instead of `self.env.context` in this case

Reply:
Agh, sorry, I have already merged it.